src/test/run-pass/exhaustive_without_never_type.rs

```diff
@@ -0,0 +1,8 @@
+#![feature(exhaustive_patterns)]
+// see issue #51221
```

Comment:
This test can be `// check-pass`.

Also please move it to some appropriate location in `src/test/ui/` (e.g. with other tests for `exhaustive_patterns`) and also reference the issue number in the test name.

Reply:
@Centril  any idea for a place? Maybe https://github.com/rust-lang/rust/tree/master/src/test/ui/uninhabited ?

Reply:
Yeah, that seems like a decent place.